roofit/hs3/inc/RooFitHS3/RooJSONFactoryWSTool.h

```diff
@@ -17,31 +18,46 @@ class RooWorkspace;
 class TH1;
 
 namespace RooFit {
-namespace Detail {
+namespace Experimental {
 class JSONNode;
 }
 } // namespace RooFit
 
-class RooJSONFactoryWSTool {
+class RooJSONFactoryWSTool : public TObject {
```

Comment:
```suggestion
class RooJSONFactoryWSTool {
```
We don't need to inherit from TObject just for the message logging. We can also use it without. In that case there is no object name in the log, but this doesn't matter for this tool because there is usually only one instance created that causes the log.